### ISSUE TITLE
i am working on a project using Laravel / vuejs vite and i have a problem in IconBase.js in ant-design-vue package and exactly on :

### ISSUE THREAD

**Issue by essaberYahya** [vueComponent/ant-design-vue#7603]
- [ ] I have searched the [issues](https://github.com/vueComponent/ant-design-vue/issues)         of this repository and believe that this is not a duplicate.
        
        
### Version
4.2.1

### Environment
laravel vuejs vite

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/sandbox/2wpk21kzvr)

### Steps to reproduce

i am working on a project using Laravel / vuejs vite and i have a problem in IconBase.js in ant-design-vue package and exactly on :

var twoToneColorPalette = reactive({ primaryColor: '#333', secondaryColor: '#E6E6E6', calculated: false });

when this code call reactive function "Default vuejs function" , it's return this exception:

reactivity.esm-bundler.js:927 Uncaught TypeError: isObject is not a function
at createReactiveObject (reactivity.esm-bundler.js:927:8)
at reactive (reactivity.esm-bundler.js:891:10)
at IconBase.js:13:27
Keep in mind that when i run npm run build the app working fine but when i run npm run dev it's generate this error . Any help please !!

Evreything without result

### What is expected?
Expectation is to work without any problem in production and devlopement

### What is actually happening?
i am working on a project using Laravel / vuejs vite and i have a problem in IconBase.js in ant-design-vue package and exactly on :

var twoToneColorPalette = reactive({ primaryColor: '#333', secondaryColor: '#E6E6E6', calculated: false });

when this code call reactive function "Default vuejs function" , it's return this exception:

reactivity.esm-bundler.js:927 Uncaught TypeError: isObject is not a function
at createReactiveObject (reactivity.esm-bundler.js:927:8)
at reactive (reactivity.esm-bundler.js:891:10)
at IconBase.js:13:27
Keep in mind that when i run npm run build the app working fine but when i run npm run dev it's generate this error . Any help please !!

Evreything without result

**Comment by cc-hearts** [comment]
Could you share a reproducible example of the code?

**Comment by essaberYahya** [comment]
There's no reproducible example of the code? because when i run npm run dev by vite script and the app try to import the ant-design-vue and especially iconbase.js file, it's generate this error : 

reactivity.esm-bundler.js:927 Uncaught TypeError: isObject is not a function
at createReactiveObject (reactivity.esm-bundler.js:927:8)
at reactive (reactivity.esm-bundler.js:891:10)
at IconBase.js:13:27

so basically it's during ant-design-vue importation .

**Comment by vivian520121** [comment]
Have you solved it? I also encountered the same problem

**Comment by essaberYahya** [comment]
> Have you solved it? I also encountered the same problem

Unfortunately no, i am still trying .

**Comment by vivian520121** [comment]
I found the problem because I imported a plugin @kangc/v-md-editor, maybe it's because of plugin conflicts

**Comment by mengdongl** [comment]
I'm having the same problem，in my case, I changed ant-design-vue to import after vue import, and it worked.
`
import { createApp } from 'vue';
import Antd from 'ant-design-vue';
`

**Comment by lzy549436494** [comment]
Hello, has the problem been solved? I also encountered the same problem.

**Comment by lnbiuc** [comment]
> I had the same problem，in my case, I changed ant-design-vue to import after vue import, and it worked. `import { createApp } from 'vue'; import Antd from 'ant-design-vue';`

solved my problem
